doc/Advanced-configuration.md

```diff
@@ -319,6 +319,31 @@ An example configuration can look as follows:
 `{odbc_server, {pgsql, "localhost", "username", "database", "pass",
                [{ssl, required}, {ssl_opts, [{verify, verify_peer}, {cacertfile, "path/to/cacert.pem"}]}]}}.`
 
+### ODBC SSL connection setup
+
+If you've configured MongooseIM to use ODBC driver, i.e. you've provided ODBC connection string to `odbc_server` option, e.g.
```

Comment:
If you've configured MongooseIM to use the ODBC driver, i.e. you've provided an ODBC connection string to `odbc_server` option, e.g.